cypress/integration/token-list/index.spec.ts

```diff
@@ -20,6 +20,12 @@ describe('Token list filter', function () {
     cy.get('#border-colors').should('not.exist');
   });
 
+  it('shows empty state', () => {
+    cy.get('input[name="tokens-filter"]').type('abc');
+    cy.get('[data-paste-element="CARD"]').should('exist');
```

Comment:
suggestion(non-blocking): use of `data-testid` as a test identifier.

By targeting a generic selector like `[data-paste-element="CARD"]` you're in effect targeting any card on the page. If someone were to add another card to the page, this test might break. To be more explicit, adding a test id to your target will guarantee your test is looking at only the thing you are trying to test

Reply:
not sure how to use `data-testid` in these cypress tests since I'm not rendering anything in the test, but would adding a `data-cy` to the card and checking for that be just as good?

`cy.get('[data-cy="tokens-empty-state"]').should('exist');`

Reply:
Yup, that's probably the cypress equivalent 👍